crates/uv-workspace/src/workspace.rs

```diff
@@ -38,6 +51,27 @@ impl Workspace {
     }
 }
 
+/// Returns the path to the global configuration directory.
+///
+/// This is similar to the `config_dir()` returned by the `dirs` crate, but it uses the
```

Comment:
I'm somewhat hesitant to diverge from the rust standard of using `dirs` here, can we file an upstream issue and link to the github discussion for this?

Reply:
You should chime in here: https://github.com/astral-sh/ruff/issues/10739